bin/redis_connections/constants.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class RedisDatabasesDefaultsEnum(int, Enum):
```

Comment:
Good idea with enum, I would just simplify the name to `RedisDatabasesEnum`

Reply:
renamed